app/renderer/task_config.js

```diff
@@ -9,74 +9,83 @@ const key = "Z4xu6Nzj";
 
 
 const defaultTasks = [{
```

Comment:
I think you did a beautify in this file with tab=2 spaces, in this project we are using tab=4 spaces

Reply:
Because of this I'm not completely sure if you did another change to this file

Reply:
Um... I think it was emacs... I've a plugin that automatic indent the code if selec all and push tab and maybe I've done that by mistake.

I didn't change nothing here... sorry D:

Reply:
Not sure if I've propertly removed this file from the commit D: